dataclients/kubernetes/testdata/routegroups/convert/failing-filter.log

```diff
@@ -1 +1 @@
-\[routegroup\] parse failed after token foo, last route id: , position 8: syntax error
+\[routegroup\] parse failed after token foo, last route id: , position 3: syntax error
```

Comment:
last route id was added by https://github.com/zalando/skipper/pull/1885
I think we should not add it to the error message if it is empty, will fix this separately.

Reply:
https://github.com/zalando/skipper/pull/2875